### PR TITLE
Update Solidity docs version to 0.4.1

### PR DESCRIPTION
**Pull request by nicksavers**
The docs version on in the browser title (and also Google search results) still mentions 0.2.0

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -56,9 +56,9 @@ def setup(sphinx):
 # built documents.
 #
 # The short X.Y version.
-version = '0.2.0'
+version = '0.4.1'
```
Comment (chriseth) [inline docs/conf.py:59]: As this is an actual pyhon file, we could also try to read the version from `../CMakeLists.txt`

Comment (pirapira) [reply docs/conf.py:59]: I crated #1088 for this.

@nicksavers if you are already working on this, let others know by assigning #1088 to yourself.

**Comment by pirapira** [comment]
I think it's good to merge this early.  Whenever I search for the solidity documentation, I see "0.2.0" in the search engine results.

**Comment by pirapira** [comment]
@chriseth I can label but cannot merge PRs currently.

**Pull request merged**